src/main.cpp

```diff
@@ -38,14 +39,18 @@ struct options {
 
 static void help(FILE *fp)
 {
-    fprintf(fp,
-            "%s [OPTIONS...]\n\n"
-            "  -c --conf-file                .conf file with configurations for "
-            "camera-streaming-daemon.\n"
-            "  -d --conf-dir <dir>          Directory where to look for .conf files overriding\n"
-            "                               default conf file.\n"
-            "  -h --help                    Print this message\n",
-            program_invocation_short_name);
+    fprintf(
+        fp,
+        "%s [OPTIONS...]\n\n"
+        "  -c --conf-file                   .conf file with configurations for "
+        "camera-streaming-daemon.\n"
+        "  -d --conf-dir <dir>              Directory where to look for .conf files overriding\n"
+        "                                   default conf file.\n"
+        "  -g --debug-log-level <level>     Set debug log level. Levels are\n"
+        "                                   <error|warning|notice|info|debug>\n"
+        "  -v --verbose                     Verbose. Same as --debug-log-level=debug\n"
+        "  -h --help                        Print this message\n",
+        program_invocation_short_name);
```

Comment:
Unrelated changes should be avoided. And following the same coding style as in our other projects would help. Also, there's room for improvement in the script to check coding style that was started on mavlink-router.